tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_3d.py

```diff
@@ -0,0 +1,254 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import OrderedDict
+
+import cv2
+import numpy as np
+
+from ..adapters import Adapter
+from ..adapters.pose_estimation import HumanPoseAdapter
+from ..config import ConfigValidator, StringField
+from ..representation import PoseEstimationPrediction, PoseEstimation3dPrediction
+
+AVG_PERSON_HEIGHT = 180
```

Comment:
Can this value be different in another cases?
If yes, possibly, it makes sense to make this constant is configurable parameter of adapter with default value.

Reply:
I'm not aware of any.

Reply:
ok, let's leave it as it is until we need to change something